src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/sections/helpers.ts

```diff
@@ -0,0 +1,40 @@
+export const getMatchedSpeciesList = async (searchTerm: string, props: any) => {
```

Comment:
why is this function async?

Reply:
Missed to remove it. Initially I had it loading the data directly from the rest API.